### PR TITLE
MC-11019: Added associate public ip

### PR DESCRIPTION
**Pull request by franzpgarcia**

### CODE REVIEW

File: source/includes/azure/_public_ip_addresses.md [hunk 1]
```diff
@@ -143,3 +143,31 @@ Optional | &nbsp;
...
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/publicipaddresses?operation=associate</code>
+
+Associate a public IP address in a given [environment](#administration-environments) to a network interface
+
+Required | &nbsp;
+------- | -----------
+`networkInterfaceId` <br/>*string* | Id of the network interface associated to an instance. Th network interface must be in the same region as the public IP address.
```
Comment (fyrnaga) [inline source/includes/azure/_public_ip_addresses.md:171]: Our APIs are not returning this information. We should expose something to allow client with our api to use it.

Comment (franzpgarcia) [reply source/includes/azure/_public_ip_addresses.md:171]: You mean to add a list network interface? or to add to the list instance?

Comment (fyrnaga) [reply source/includes/azure/_public_ip_addresses.md:171]: I guess adding them to the instance list would be fine for now

Comment (franzpgarcia) [reply source/includes/azure/_public_ip_addresses.md:171]: I'll open another story.

Comment (fyrnaga) [reply source/includes/azure/_public_ip_addresses.md:171]: ok

**Review by fyrnaga — APPROVED** [review]
LGTM

**Review by GuillaumeVialle — APPROVED** [review]
LGTM

**Pull request merged**